pkg/controller/operator.go

```diff
@@ -69,10 +69,18 @@ func (s *Support) Run(controller *controllercmd.ControllerContext) error {
 	}
 
 	// these are gathering clients
+	gatherProtoKubeConfig := rest.CopyConfig(controller.ProtoKubeConfig)
+	if len(s.Impersonate) > 0 {
+		gatherProtoKubeConfig.Impersonate.UserName = s.Impersonate
+	}
 	gatherCRKubeConfig := rest.CopyConfig(controller.KubeConfig)
```

Comment:
What does `CR` stand for here?

Reply:
Custom resource, you can’t use proto with crds yet from the same client

Reply:
Ty